gen/definitions/wireless_ssid.yaml

```diff
@@ -0,0 +1,524 @@
+name: Wireless SSID
+rest_endpoint: /networks/%v/wireless/ssids
+data_source_name_query: true
```

Comment:
I don't think this makes sense here, as we need the number for the path anyway.

Reply:
this could be implemented because there is a [GET endpoint that lists all the SSIDs](https://developer.cisco.com/meraki/api-v1/get-network-wireless-ssids/). but that would require the data source to have a different set of mandatory fields than the resource, so this would have to be manually implemented. removing for now, maybe we'll add this functionality later. i don't think this will that relevant in production anyways because all the SSIDs are explicitly fixed to their numbers (there is no POST endpoints, but rather a fixed number of SSID slots that could be edited with PUT)